openedx/core/djangoapps/user_api/accounts/tests/test_api.py

```diff
@@ -354,6 +361,21 @@ def test_create_account(self):
             'extended_profile': [],
         })
 
+    @override_waffle_flag(PASSWORD_UNICODE_NORMALIZE, active=True)
+    def test_normalize_password(self):
+        # Test password normalization on account creation.
+        create_account(self.USERNAME, unicodedata.normalize('NFKD', u'Ṗŕệṿïệẅ Ṯệẍt'), self.EMAIL)
+
+        # Retrieve the account settings
+        user = User.objects.get(username=self.USERNAME)
+        request = RequestFactory().get("/api/user/v1/accounts/")
+        request.user = user
+
+        salt_val = user.password.split('$')[1]
+
+        user_hashed_password = make_password(unicodedata.normalize('NFKC', u'Ṗŕệṿïệẅ Ṯệẍt'), salt_val)
+        self.assertEqual(user_hashed_password, user.password)
```

Comment:
This feels very similar to the other test in this PR that checks the salt and all that. Is it a different code path?

Reply:
Yes, it is a different code path. Yay platform :)